lightning/src/ln/async_signer_tests.rs

```diff
@@ -10,18 +10,21 @@
 //! Tests for asynchronous signing. These tests verify that the channel state machine behaves
 //! properly with a signer implementation that asynchronously derives signatures.
 
+use std::collections::HashSet;
```

Comment:
nit: in most files we have a `use crate::prelude::*;`, which I believe would avoid this